### DISCUSSION TITLE
Migration error in Firefly

### DISCUSSION THREAD

**Discussion by KK887788** [iotaledger/firefly#7749] [Help & Support]
Dear all,

When trying to migrate my MIOTAS into the firefly wallet, I only get the report that an unknown error occured. What can I do now?

**Comment by begonaalvarezd** [comment]
When you say migrate you mean coming from a Trinity Legacy seed? Or what do you mean?

**Reply by KK887788** [reply]
Yea. last time I checked on the Miotas was on the trinity wallet platform. After the integration of the firefly wallet I wasn't able to restore my Miotas so far because there is always an error when I enter the seed

**Reply by begonaalvarezd** [reply]
The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again.
We will provide a migration tool to resume legacy migrations, but its not finished yet

Will close the issue because Firefly is not intended to provide migration anymore, but it will be a different tool. The Foundation will notify it when the tool is ready 🌷
